### ISSUE TITLE
Bitcoin Wallet-Identity Manager-Voltea la imagen tomada desde camara

### ISSUE THREAD

**Issue by danielaporras** [Fermat-ORG/beta-testing-program#748]
#### Nombre pantalla
Identity Manager

### Nombre de la Aplicación
Bitcoin Wallet 

-----

##### Reporte de error: 

   *Al seleecionar la opcion Camera en la pantalla de (Choose mode) para crear el perfil en la bitcoin wallet la foto tomada se voltea 90º, aunque es poco visible por el error ya reportado (721).

 *Agregar las capture del error en el emulador o teléfono donde se produjo. 
Capture 1

![screenshot_5](https://cloud.githubusercontent.com/assets/19656830/17569039/76a1f2a8-5f14-11e6-9dc7-094ac61565d8.png)
Capture 2

![screenshot_6](https://cloud.githubusercontent.com/assets/19656830/17569043/7de9c4fa-5f14-11e6-92c8-bb2a7356a121.png)

##### Pasos exactos para reproducir el error:

1.Entre a la bitcoin wallet para crear el perfil.
2.Al seleccionar la opcion imagen desde camera y tomar la foto.
3.Pasar a la pantalla donde se arregla el tamaño de la imagen (capture2) se voltea 90º a la derecha.

##### Inserta Archivo aLogcat: 

 
[Google Nexus 4 - 4.1.1 - API 16 - 768x1280-logcat.txt](https://github.com/Fermat-ORG/beta-testing-program/files/412034/Google.Nexus.4.-.4.1.1.-.API.16.-.768x1280-logcat.txt)

 
  
------

#### Dispositivo desde donde se realizo la prueba

- [x] Emulador

##### Screenshot o Capture de pantalla:
![screenshot_1](https://cloud.githubusercontent.com/assets/19656830/17569364/db98a660-5f15-11e6-93ad-593dac6a5e96.png)

@AdryCanonico

**Comment by IsbetVR** [comment]
@nattyco

**Comment by nattyco** [comment]
@danielaporras @IsbetVR  @MaviDavsa  Por lo que veo de este reporte que es comun a CBP la prueba se hiso solo con el emulador, hay que probar con los celulares y tomar otras fotos para descartar que sea algo que hace el emulador, entregar la imagen rotada.

**Comment by danielaporras** [comment]
@nattyco claro, a @AdryCanonico  le paso en el telefono fisico, esperemos a que ella comente y suba sus capture.

**Comment by AdryCanonico** [comment]
En la version 21.9 de CBP Ya no sucede el error.

![screenshot_2016-08-11-11-25-57](https://cloud.githubusercontent.com/assets/18584168/17594192/6b5e37e2-5fb6-11e6-9157-68d780e401f8.png)

http://www.smart-gsm.com/moviles/samsung-galaxy-avant


![samsung avant](https://cloud.githubusercontent.com/assets/18584168/17594250/9854b5fa-5fb6-11e6-8bca-ad200dabb18d.png)

**Comment by JeanFranco498** [comment]
Version 21.9 Sigue pasando 

![image](https://cloud.githubusercontent.com/assets/19807906/17621954/38f3233a-6065-11e6-952e-2c46651cae5c.png)

![screenshot_20](https://cloud.githubusercontent.com/assets/19807906/17621958/40d9fda8-6065-11e6-877b-8a74a35d0d23.jpg)

**Comment by GeronimoVierma** [comment]
En la versión 22.7 de CBP me ocurre este error desde mi teléfono físico, al momento de tomar la foto esta se voltea 90º

![screenshot_2016-08-26-09-06-17](https://cloud.githubusercontent.com/assets/19883115/18007070/83719638-6b70-11e6-95f5-001905716a28.png)

![screenshot_2016-07-14-18-31-45](https://cloud.githubusercontent.com/assets/19883115/18007065/7e9d4d0a-6b70-11e6-870e-0f665c1ab72e.png)

**Comment by IsbetVR** [comment]
Se reabre el caso

**Comment by nattyco** [comment]
@IsbetVR  @GeronimoVierma Este caso ya se aclaro que la imagen la rota la camara, y es un tema de hardware del celular no tiene nada que ver con la aplicacion, por eso fue que se agrego en las identities la opcion de rotar y recortar la foto para que el usuario la acomode.

**Comment by yolandayancel** [comment]
Prueba 12-06-16
Caso 748 CCp  v 3.3.5
Este error se soluciono en mi celular:
![screenshot_2016-09-12-08-10-52](https://cloud.githubusercontent.com/assets/17283358/18436546/f8773114-78c6-11e6-9a0e-b3a20a82f202.png)
![screenshot_2016-09-12-08-26-53](https://cloud.githubusercontent.com/assets/17283358/18436591/2fbcea24-78c7-11e6-96f2-1cc582ab32dc.png)

![screenshot_2016-09-06-15-20-35](https://cloud.githubusercontent.com/assets/17283358/18436569/1bc33faa-78c7-11e6-8ed6-5cb725e58654.png)

**Comment by arosiopr** [comment]
Solucionado en la versión 3.3.5 de CCP @IsbetVR 

![screenshot_2016-09-12-08-16-00](https://cloud.githubusercontent.com/assets/5313326/18436661/887d8092-78c7-11e6-82e9-e906e3972041.png)

![screenshot_2016-09-12-08-16-09](https://cloud.githubusercontent.com/assets/5313326/18436664/8a6cc85e-78c7-11e6-9b42-2f460ae3518a.png)

![alcatel](https://cloud.githubusercontent.com/assets/5313326/18436680/9bf3d73e-78c7-11e6-8054-eac7041f7f39.png)
